.changes/unreleased/tinymce-TINY-11100-2024-08-05.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Deleting a selection in a list element would sometimes prevent the `input` event
+  from firing
```

Apparently fire is a bad word.

```suggestion
  from being dispatched.
```